NLibrary/Classes/DashboardHomeMVVM/DashboardViewModel.swift

```diff
@@ -97,7 +98,7 @@ public class DashboardViewModel: DashboardViewModelProtocol {
     }
 
     public func getTrending() {
-        self.repo?.getTrending() { [weak self] result in
+        let x = self.repo?.getTrending() { [weak self] result in
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [When using trailing closures, empty parentheses should be avoided after the method call.](https://app.codacy.com/manual/NathanDVT/NLibrary/pullRequest?prid=5376515)